### PR TITLE
Unittest logged in index

### PR DESCRIPTION
**Pull request by sjanssen2**

### CODE REVIEW

**Comment by wasade** [comment]
Some ci failures

File: knimin/lib/tests/test_util.py [hunk 1]
```diff
@@ -66,6 +69,7 @@ def test_categorize_age(self):
         self.assertEqual('70+', categorize_age(122))
         self.assertEqual('Unspecified', categorize_age(123))
         self.assertEqual('Unspecified', categorize_age(123564))
+        self.assertEqual('Unspecified', 'Unspecified')
```
Comment (wasade) [inline knimin/lib/tests/test_util.py]: ?

Comment (wasade) [inline knimin/lib/tests/test_util.py]: Looks like this is missing a call to categorize_age

File: knimin/lib/tests/test_util.py [hunk 2]
```diff
@@ -97,7 +102,99 @@ def test_categorize_bmi(self):
...
+                    ["000033914", "000033915", "000033916", "000033917",
+                     "100033914", "100033915", "100033916", "100033917"]]]
+        result = get_printout_data(kitinfo)
+        for kit in kitinfo:
+            self.assertIn("Sample Barcodes:\t%s" % ', '.join(kit[3][:5]),
+                          result)
+            self.assertIn("\t\t\t%s" % ', '.join(kit[3][5:]), result)
+
+    def test_fetch_url(self):
+        # test unkown address
```
Comment (wasade) [inline knimin/lib/tests/test_util.py]: Unknown

File: knimin/lib/tests/test_util.py (hunk 1 above)
Comment (sjanssen2) [inline knimin/lib/tests/test_util.py]: ups. I added categorize_age().

File: knimin/lib/tests/test_util.py (hunk 2 above)
Comment (sjanssen2) [inline knimin/lib/tests/test_util.py]: fixed typo

**Pull request merged**